src/hotspot/share/runtime/stackWatermark.cpp

```diff
@@ -77,14 +77,12 @@ void StackWatermarkFramesIterator::set_watermark(uintptr_t sp) {
 // frame processing code, such as creating random handles even though there
 // is no safepoint to protect against, and fiddling around with exceptions.
 class StackWatermarkProcessingMark {
-  ResetNoHandleMark _rnhm;
```

Comment:
Since you're removing the ResetNoHandleMark, you should revisit
the comment block for the class. The entire sentence to revisit is:
```
// ... It is mostly
// due to problems that we might want to eventually clean up inside of the
// frame processing code, such as creating random handles even though there
// is no safepoint to protect against, and fiddling around with exceptions.
```